blog/_posts/2015-12-2-includeos-is-now-open-source.md

```diff
@@ -0,0 +1,38 @@
+#IncludeOS is now free and open source
+
+We've finally opened the lid on [IncludeOS](http://www.includeos.org), just in time for the [IEEE CloudCom](http://2015.cloudcom.org/) paper presentation today. We've done quite a lot of work since the paper was written, so here's an update on what IncludeOS is now, and what you can expect in the near future. 
+
+## It's a bit like a java-vm, but for x86 C++
+A Java Virtual Machine is a portable language runtime environment. Java becomes portable across hardware architectures and operating systems, because it uses a common instruction set. Once you've started a Java program, you can't log into it (unless your program itself provides the facilities), and you can't boot up any other programs inside it.
+
+IncludeOS is like a language runtime for C++ programs, compiled into the common x86 instruction set. This has the obvious advantage of removing one layer of abstraction, compared to Java; with hardware virtualization the code will execute directly on the CPU. Like with a Java program, and other unikernels, an IncludeOS service is a program, with the language runtime attached. It's called IncludeOS because you start by saying `#include <os>` and whatever your service needs from the operating system will be included into the virtual machine. 
```

Comment:
Any restrictions on the x86 instructions supported?
(Is this "common" in the sense of "commonplace" or "common" in the sense of "a shared subset"?)